neps/nep-0330.md

```diff
@@ -51,7 +59,17 @@ As an example, say there was an NFT contract deployed on-chain which was current
 ```ts
 type ContractSourceMetadata = {
   version: "39f2d2646f2f60e18ab53337501370dc02a5661c"
-  link: "https://github.com/near-examples/nft-tutorial"
+  link: "https://github.com/near-examples/nft-tutorial",
+  standards: [
+    {
+        standard: "nep171", 
+        version: "1.0.2"
+    },
+    {
+        standard: "nep177", 
+        version: "2.0.1"
+    }
+  ]
```

Comment:
this is on chain data, can we make it shorter, or maybe just pairs? so:
```
[
  [ "nep171", "1.0.2"],
  [ "nep177", "2.0.1"]
]
```

Reply:
@austinabell, you are right. Most probably this will be hardcoded. 
But even for the wire format, I'm not sure if it makes sense to send objects. Maybe it's small enough and not relevant?

Reply:
What we would generally be talking about here is off-chain costs, as it's unlikely it would be used in a transaction. Is there a use case for this other than someone performing view calls to query for each contract type?

It seems to me to be a little unnecessary to optimize this for encoded size at the cost of readability/UX when even other standards that are used for cross-contract interactions were not optimized, for whatever reason